app/js/lib/server/webpush.js

```diff
@@ -30,56 +30,31 @@ export default class WebPush extends EventDispatcher {
     navigator.serviceWorker.addEventListener('message',
       this[p.listenForMessages].bind(this));
 
-    const settings = this[p.settings];
-    if (settings.pushEndpoint && settings.pushPubKey && settings.pushAuth &&
-        !resubscribe) {
-      return Promise.resolve();
-    }
-
-    return navigator.serviceWorker.ready
-      .then((reg) => reg.pushManager.subscribe({ userVisibleOnly: true }))
-      .then((subscription) => {
-        const endpoint = subscription.endpoint;
-        const key = subscription.getKey ? subscription.getKey('p256dh') : '';
-        const auth = subscription.getKey ? subscription.getKey('auth') : '';
-        settings.pushEndpoint = endpoint;
-        settings.pushPubKey = btoa(String.fromCharCode.apply(null,
-          new Uint8Array(key)));
-        settings.pushAuth = btoa(String.fromCharCode.apply(null,
-          new Uint8Array(auth)));
-
-        // Send push information to the server.
-        // @todo We will need some library to write taxonomy messages.
-        const pushConfigurationMsg = [[
-            [{ id: 'channel:subscribe.webpush@link.mozilla.org' }],
-            {
-              subscriptions: [{
-                public_key: settings.pushPubKey,
-                push_uri: settings.pushEndpoint,
-                auth: settings.pushAuth,
-              }],
-            },
-          ]];
-
-        return this[p.api].put('channels/set', pushConfigurationMsg);
-      })
-      .then(() => {
-        // Setup some common push resources.
-        const pushResourcesMsg = [[
-          [{ id: 'channel:resource.webpush@link.mozilla.org' }],
-          { resources: ['res1'] },
-        ]];
+    return navigator.serviceWorker.ready.then((reg) => {
```

Comment:
Please take a look at `CONTRIBUTING.md` on the project coding style. `.then(...` are usually  placed on a new line and indented.